README.adoc

```diff
@@ -323,6 +323,29 @@ foo&. bar
 foo&.bar
 ----
 
+=== Safe navigation
+
+Avoid long chains of `&.`. Replace with `.` if they introduce unnecessary conditional logic. 
```

Comment:
What does it mean "if they introduce unnecessary conditional logic"?

Reply:
Updated the sentence, I mean to say if introducing the safe operator adds unnecessary conditional logic we should use other methods such as delegation.